src/fund.rs

```diff
@@ -0,0 +1,94 @@
+use ethcore::{
+    self,
+    state::{CleanupMode, State},
+};
+use ethereum_types::{Address, H256, U256};
+use oasis_ethwasi_runtime_common::parity::NullBackend;
+extern crate chrono;
+use chrono::{DateTime, Datelike, NaiveDateTime, Utc};
+extern crate date_time;
+use byteorder::{BigEndian, ReadBytesExt};
+use date_time::month_tuple::MonthTuple;
+use std::{convert::TryInto, str::FromStr};
+
+fn timestamp_to_mt(timestamp: i64) -> MonthTuple {
+    let dt = DateTime::<Utc>::from_utc(NaiveDateTime::from_timestamp(timestamp, 0), Utc);
+    return MonthTuple::new(
+        dt.year().try_into().unwrap(),
+        dt.month().try_into().unwrap(),
+    )
+    .unwrap();
+}
+
+pub struct FundManager;
+
+impl FundManager {
+    /// Unlock address.
+    const BENEFICIARY: &'static str = "7110316b618d20d0c44728ac2a3d683536ea682b";
+    /// Adjust period.
+    const PERIOD: u32 = 20;
+    // Factor
+    const FACTOR: f32 = 0.5;
+    /// Total Amount.
+    const TOTAL_AMOUNT: u32 = 21000000;
```

Comment:
Should we use `wei` as unit here?

Reply:
After using `wei` as unit, we should use `u256` here.

Reply:
done.